keyboards/system76/launch_1/config.h

```diff
@@ -18,94 +18,92 @@
 #pragma once
 
 #include "config_common.h"
+#ifndef __ASSEMBLER__
+#include "eeconfig.h"
+#include "eeprom.h"
+#endif
 
 // Key matrix size
 #define MATRIX_ROWS 6
 #define MATRIX_COLS 14
 
-/*
- * Key matrix pins
- * ROWS: AVR pins used for rows, top to bottom
- * COLS: AVR pins used for columns, left to right
- */
+// Key matrix pins
+// ROWS: AVR pins used for rows, top to bottom
+// COLS: AVR pins used for columns, left to right
 #define MATRIX_ROW_PINS { F0, F1, F4, F5, F6, F7 }
 #define MATRIX_COL_PINS { D7, C7, C6, B6, B5, B4, D6, D4, E6, D5, D3, D2, B7, B0 }
 
-/*
- * Diode Direction
- * COL2ROW = COL => Anode (+), ROW => Cathode (-)
- * ROW2COL = ROW => Anode (+), COL => Cathode (-)
- */
+// Matrix diode direction
+// COL2ROW: COL => Anode (+), ROW => Cathode (-)
+// ROW2COL: ROW => Anode (+), COL => Cathode (-)
 #define DIODE_DIRECTION COL2ROW
 
-// Set 0 if debouncing isn't needed
+// Delay when reading the value of the pin; set to `0' if debouncing is not needed
 #define DEBOUNCE 5
 
+// Disable mechanical locking support (`KC_LCAP', `KC_LNUM', or `KC_LSCR')
+#undef LOCKING_SUPPORT_ENABLE
+
+// Disable switch state consistency with keyboard LED state
+#undef LOCKING_RESYNC_ENABLE
+
+// RGB matrix
 #ifdef RGB_MATRIX_ENABLE
-#    define RGB_DI_PIN E2
-#    define RGB_MATRIX_LED_COUNT 84
-#    define RGB_MATRIX_KEYPRESSES                                      // Reacts to keypresses
-// #    define RGB_MATRIX_KEYRELEASES                                     // Reacts to keyreleases (instead of keypresses)
-// #    define RGB_MATRIX_FRAMEBUFFER_EFFECTS                             // Enables framebuffer effects
-#    define RGB_DISABLE_WHEN_USB_SUSPENDED                             // Turns off effects when suspended
+#define RGB_DI_PIN E2
```

Comment:
why is the formatting being undone? 

Running `qmk format-c` on these files will undo the changes you've added here.

Reply:
Reran ClangFormat on `config.h`.